orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/CompoundExecutionOperator.java

```diff
@@ -77,21 +79,32 @@ public void resume(
       @Nullable String user,
       @NonNull Boolean ignoreCurrentStatus) {
     doInternal(
-        () -> runner.unpause(repository.retrieve(executionType, executionId)),
+        (Execution execution) -> runner.unpause(execution),
         () -> repository.resume(executionType, executionId, user, ignoreCurrentStatus),
         "resume",
         executionType,
         executionId);
   }
 
   private void doInternal(
-      Runnable runnerAction,
+      Consumer<Execution> runnerAction,
       Runnable repositoryAction,
       String action,
       ExecutionType executionType,
       String executionId) {
     try {
-      runWithRetries(runnerAction);
+      runWithRetries(
+          () -> {
+            Execution execution = repository.retrieve(executionType, executionId);
+            if (repository.handlesPartition(execution.getPartition())) {
+              runnerAction.accept(execution);
+            } else {
+              log.warn(
```

Comment:
this will be happening a lot so prob not worth a `warn`?

Reply:
esp since we will get an interlink publish log shortly after

Reply:
handshake deal, making it an `info`